src/main/java/com/google/api/codegen/transformer/php/PhpGapicSamplesTransformer.java

```diff
@@ -96,12 +96,74 @@ public PhpGapicSamplesTransformer(
                 .libraryMethod(
                     method.toBuilder().samples(Collections.singletonList(methodSample)).build())
                 .gapicPackageName(namer.getGapicPackageName(packageConfig.packageName()))
+                .extraInfo(
+                    PhpSampleExtraInfo.newBuilder()
+                        .hasDefaultServiceScopes(
+                            context.getInterfaceConfig().hasDefaultServiceScopes())
+                        .hasDefaultServiceAddress(
+                            context.getInterfaceConfig().hasDefaultServiceAddress())
+                        .build())
                 .build());
       }
     }
     return viewModels.build();
   }
 
+  public static class PhpSampleExtraInfo extends DynamicLangSampleView.SampleExtraInfo {
```

Comment:
Can't you use Autovalue for this class (for the `has*` methods)? I would expect  it would be possible.

Alternatively, you can forgo the Builder pattern entirely and either:
1. have a class with accessors and a param constructor instead of a builder
2. have a class with public fields instead of accessors and a param constructor instead of a builder

I'd probably consider 1 for simplicity. But if you want to leave as is, the code's correct too, obviously.